ui/main/src/app/business/view/dashboard/dashboard.view.spec.ts

```diff
@@ -60,13 +69,27 @@ describe('Dashboard', () => {
             null,
             opfabLoggerService
         );
+
+        TestBed.configureTestingModule({
+            providers: [
+                {provide: ConfigServer, useClass: ConfigServerMock},
+                {provide: RemoteLoggerServer, useValue: null},
+                {provide: ProcessServer, useClass: ProcessServerMock},
+                {provide: AcknowledgeServer, useValue: null},
+                LightCardsStoreService,
+                {provide: EntitiesServer, useValue: null},
+                {provide: UserServer, useValue: null},
+            ]
+        });
+
```

Comment:
Avoid to use testingBed to be independant of angular 

Should make the job : 

```
        const entitiesService = new EntitiesService(opfabLoggerService,null,new AlertMessageService());
        const userPermissionService = new UserPermissionsService(entitiesService,processesService);
        acknowledgeService = new AcknowledgeService(null,userPermissionService,userService,processesService,entitiesService);
```

Reply:
Done.